.gitignore

```diff
@@ -1,4 +1,16 @@
 node_modules/
 dist/
-.env
+.env*
+!.env.example
```

Comment:
Would be fantastic to add that to the generated .gitignore files as well (`templates/base/.gitignore`)